### PR TITLE
Support configuration of EBS volumes for EMR clusters

### PR DESCRIPTION
**Pull request by mosche**
Here's an example how to attach EBS volumes to cluster nodes:
https://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/emrcluster-example-ebs.html

The relevant documentation parts of the : 
https://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-object-emrcluster.html
However, details on `EbsConfiguration` are missing in the developer guide :(

### EbsBlockDeviceConfig
The configuration of a requested Amazon EBS block device associated with the instance group. Includes a specified number of volumes that will be associated with each instance in the instance group. Includes volumesPerInstance and volumeSpecification, where:
- volumesPerInstance is the number of EBS volumes with a specific volume configuration that will be associated with each instance in the instance group.
- volumeSpecification is the Amazon EBS volume specifications, such as volume type, IOPS, and size in Gigibytes (GiB) that will be requested for the EBS volume attached to an EC2 instance in the Amazon EMR cluster.

### VolumeSpecification
The Amazon EBS volume specifications, such as volume type, IOPS, and size in Gigibytes (GiB) that will be requested for the Amazon EBS volume attached to an Amazon EC2 instance in the Amazon EMR cluster. The node can be a core, master or task node.

The VolumeSpecification includes:
- iops() Integer. The number of I/O operations per second (IOPS) that the Amazon EBS volume supports, for example, 1000. For more information, see EBS I/O Characteristics in the Amazon EC2 User Guide for Linux Instances.
- sizeinGB(). Integer. The Amazon EBS volume size, in gibibytes (GiB), for example 500. For information about valid combinations of volume types and hard drive sizes, see EBS Volume Types in the Amazon EC2 User Guide for Linux Instances.
- volumetType. String. The Amazon EBS volume type, for example, gp2. The supported volume types include standard, gp2, io1, st1, sc1, and others. For more information, see EBS Volume Types in the Amazon EC2 User Guide for Linux Instances.

### CODE REVIEW

**Comment by mosche** [comment]
@realstraw Me again :) One thing, I noticed the project doesn't contain any code formatting guidelines or scalariform settings. That would make contributions a lot simpler :)

**Comment by realstraw** [comment]
Thanks for the PR @mosche . Yes, totally agree, we should add code formatting guidelines or scariform settings. We have something available internally, I'll get that cleaned up and make it available here.

I'll take a look at the PR later this week.

File: core/src/main/scala/com/krux/hyperion/resource/EmrConfiguration.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ case class EmrConfiguration private (
   configurations: Seq[EmrConfiguration]
 ) extends NamedPipelineObject {
 
-  type Self = EmrConfiguration
+    type Self = EmrConfiguration
```
Comment (realstraw) [inline core/src/main/scala/com/krux/hyperion/resource/EmrConfiguration.scala]: the previous indentation was correct

**Comment by mosche** [comment]
Created this issue to link it in the changelog @realstraw 
https://github.com/krux/hyperion/issues/576

**Comment by mosche** [comment]
Actually, if you wanna wait a bit, I'll build a Snapshot here and quickly run a datapipeline to test it for real

**Comment by realstraw** [comment]
Sure

**Comment by mosche** [comment]
Sorry, got distracted ... 
After adding all the pipeline objects, I successfully tested this with one of our pipelines. So ready to be merged from my side

File: build.sbt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-val hyperionVersion = "5.3.0"
+val hyperionVersion = "5.3.1"
```
Comment (realstraw) [inline build.sbt]: can you change this to 5.4.0 instead

Comment (mosche) [reply build.sbt]: right, minor you said, sorry

**Comment by mosche** [comment]
@realstraw I've changed the version to 5.4.0

**Comment by realstraw** [comment]
Thanks, @mosche ! I'm merging it

**Pull request merged**